indexing-hadoop/src/main/java/io/druid/indexer/JobHelper.java

```diff
@@ -23,7 +23,6 @@
 import com.google.common.base.Predicate;
```

Comment:
Line 718 of this file: should the URI be created with the scheme written in the loadSpec for the key 'S3Schema' if it exists? (and then default to s3n if it doesn't)